### PR TITLE
feat(magneto): #MAG-41/MAG-166 add card comments + fix card display

### PR DESCRIPTION
**Pull request by flmartineau**
Voir les PRs : 
https://github.com/CGI-OPEN-ENT-NG/magneto/pull/76
https://github.com/CGI-OPEN-ENT-NG/magneto/pull/77
https://github.com/CGI-OPEN-ENT-NG/magneto/pull/82

### CODE REVIEW

File: scss/specifics/magneto/components/containers/_board-read.scss [hunk 1]
```diff
@@ -36,6 +42,12 @@
     left: 20px;
     z-index: 1;
     color: $magneto-black;
+
+    &:before {
+      background-color: white;
```
Comment (JinDouang) [inline scss/specifics/magneto/components/containers/_board-read.scss:47]: var

File: scss/specifics/magneto/components/containers/_board.scss [hunk 2]
```diff
@@ -81,7 +82,7 @@
     }
 
     &-list {
-      min-height: 80vh;
+      min-height: 650px;
```
Comment (Sofianebernoussi) [inline scss/specifics/magneto/components/containers/_board.scss:85]: - adapte a tout les tailles d'ecrans non?

File: scss/specifics/magneto/components/directives/_card-list-item-options.scss [hunk 3]
```diff
@@ -1,9 +1,11 @@
   .card-list-item-options {
+    transform: translateX(5px) translateY(-26px);
+    z-index: 2;
     nav {
       .nav, .sub-nav {
         position: absolute;
-        top: 0;
-        left: 100%;
+        //top: 0;
```
Comment (Sofianebernoussi) [inline scss/specifics/magneto/components/directives/_card-list-item-options.scss:7]: si inutile go delete?

**Review by Sofianebernoussi — APPROVED** [review]

File: scss/specifics/magneto/components/directives/_card-list-item-options.scss (hunk 3 above)
Comment (alicedraillard) [inline scss/specifics/magneto/components/directives/_card-list-item-options.scss:7]: a supprimer ?

**Review by alicedraillard — APPROVED** [review]

File: scss/specifics/magneto/components/directives/_card-list-item-options.scss (hunk 3 above)
Comment (flmartineau) [reply scss/specifics/magneto/components/directives/_card-list-item-options.scss]: oui :)

**Pull request merged**